packages/scan/src/Scan.ts

```diff
@@ -94,6 +94,10 @@ export class Scan {
       clearTimeout(timer);
     }
 
+    if (this.done && status !== ScanStatus.DONE) {
+      throw new Error(`Scan failed with status ${status}.`);
+    }
```

Comment:
If someone stops a scan from UI, you will get an error as well. That is what you expect, right?

Reply:
Yes, as well as stop from api or by accident from test itself